RFC-005-SWAP-Basic-Bitcoin.md

```diff
@@ -143,6 +143,41 @@ For how to use this witness data to construct the redeem transaction see [BIP141
 To be notified of the redeem event, both parties MAY watch the blockchain for transactions that spend from the output and check that the witness data is in the above form.
 If Bitcoin is the `beta_ledger` (see [RFC003](./RFC-003-SWAP-Basic.md)), then the funder MUST watch for such a transaction and  extract the `secret` from its witness data and continue the protocol.
 
+#### Evaluation of the script for redeem
+
+**Given** a bitcoin script address that holds the following (hashed) information:
+1. `script code`
```

Comment:
The wording is strange. `1.` contains `2.`, `3.` & `4.`.

Reply:
As far as I understood it, the transaction contains not only the parameters but also the script to be executed. The script's hash is checked against the address to evaluate if the script given is correct and will be executed. Is that incorrect?